tests/attr/test_integrated_gradients_basic.py

```diff
@@ -253,20 +266,22 @@ def _assert_batched_tensor_multi_input(
 
     def _compute_attribution_and_evaluate(
         self,
-        model,
-        inputs,
-        baselines=None,
-        target=None,
-        additional_forward_args=None,
-        type="vanilla",
-        approximation_method="gausslegendre",
-    ):
+        model: Module,
+        inputs: TensorOrTupleOfTensors,
+        baselines: Optional[
+            Union[Tensor, int, float, Tuple[Union[Tensor, int, float], ...]]
+        ] = None,
+        target: Optional[int] = None,
+        additional_forward_args: Optional[Any] = None,
+        type: Optional[str] = "vanilla",
+        approximation_method: str = "gausslegendre",
+    ) -> Tuple[Tensor, ...]:
         r"""
             attrib_type: 'vanilla', 'smoothgrad', 'smoothgrad_sq', 'vargrad'
         """
         ig = IntegratedGradients(model)
         if not isinstance(inputs, tuple):
-            inputs = (inputs,)
+            inputs = cast(TensorOrTupleOfTensors, (inputs,))
```

Comment:
This changes behavior, since cast doesn't have any effect at runtime. Can we just leave the original here?

Reply:
Never mind, sorry, ignore above, this is fine!